firedrake/spatialindex.pyx

```diff
@@ -1,9 +1,11 @@
 from __future__ import absolute_import, print_function, division
```

Comment:
While we're here (changing this file), we can remove the `__future__` import, since we don't support Python 2 any more.